output.tf

```
@@ -0,0 +1,6 @@
+output "pdm_object_tagger_batch_config" {
```

it will probably read better when referencing if you do this:
```suggestion
output "pdm_object_tagger_batch" {
```